project.godot

```diff
@@ -17,8 +17,13 @@ config/icon="res://assets/sprites/flirty_pose.png"
 
 [autoload]
 
+StationEvent="*res://scripts/autoload/StationEvent.gd"
```

Comment:
So StationEvent is essentially the new play-through instance?

Reply:
StationEvent is basically just a group of globally accessible signals that are emitted whenever the station variables are updated so things like the ui can be updated automatically